### PR TITLE
release-01 only - Set role/rolebinding when a new ansible job cr is created outside of the ansible job operator namespace

### PR DESCRIPTION
**Pull request by mikeshng**
This is a quick port of the cluster scope operator changes to the `release-0.1` branch.

### CODE REVIEW

File: deploy/clusterrole_binding.yaml [hunk 1]
```diff
@@ -1,11 +1,12 @@
-kind: RoleBinding
+kind: ClusterRoleBinding
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
   name: tower-resource-operator
 subjects:
 - kind: ServiceAccount
   name: tower-resource-operator
+  namespace: tower-operator
```
Comment (matburt) [inline deploy/clusterrole_binding.yaml:8]: I think I had asked this question somewhere else, but I'm wondering about the implication of having a fixed namespace here.

**Review by xiangjingli — APPROVED** [review]

File: deploy/clusterrole_binding.yaml (hunk 1 above)
Comment (mikeshng) [reply deploy/clusterrole_binding.yaml:8]: It doesn’t have to be specifically that namespace. The SA can be created in any namespace (usually same namespace as the operator deployment) but this binding will need to be updated to reference that SA’s namespace. Am I right? @xiangjingli

**Comment by matburt** [comment]
I guess we're not talking about the namespace thing 🤣

File: deploy/clusterrole_binding.yaml (hunk 1 above)
Comment (xiangjingli) [reply deploy/clusterrole_binding.yaml:8]: @mikeshng  That is true.  In this case,  we specify the SA in the same operator namespace. By this way we want to clearly demonstrate  that the SA is for the operator, not for the k8s runner job.

**Pull request merged**